changelog/security/2022-01-09-weekly-updates.md

```diff
@@ -0,0 +1,3 @@
+- vim ([CVE-2022-3491](https://nvd.nist.gov/vuln/detail/CVE-2022-3491), [CVE-2022-3520](https://nvd.nist.gov/vuln/detail/CVE-2022-3520), [CVE-2022-3591](https://nvd.nist.gov/vuln/detail/CVE-2022-3591), [CVE-2022-4141](https://nvd.nist.gov/vuln/detail/CVE-2022-4141), [CVE-2022-4292](https://nvd.nist.gov/vuln/detail/CVE-2022-4292), [CVE-2022-4293](https://nvd.nist.gov/vuln/detail/CVE-2022-4293))
```

Comment:
Can you please rename `changelog/security/2022-01-09-weekly-updates.md` to `changelog/security/2023-01-09-weekly-updates.md`?

Reply:
Right. Thanks for spotting.